ClientApp/src/assets/i18n/en.json

```diff
@@ -12,6 +12,9 @@
     },
     "settings": {
       "label": "Settings"
+    },
+    "reload":{
+      "label": "Reload Folder"
```

Comment:
In French:

"label": "Recharger le dossier"

In Spanish:

"label": "Recargar carpeta"